{"author":{"id":"e970470018827b6b178556f877e6452433eeadb8d713d3499b58f62b3765b664"},"ops":[{"type":3,"author":{"id":"e970470018827b6b178556f877e6452433eeadb8d713d3499b58f62b3765b664"},"timestamp":1691795072,"metadata":{"github-id":"IC_kwDODbKJfs5j3hxZ","github-url":"https://github.com/vmiklos/plees-tracker/issues/407#issuecomment-1675500633"},"nonce":"UuJ5ppavj3X7B6xL4XspqiYWQhw=","message":"This is ArrowOS 13.1(TQ3A.230705.001) on lisa device. Right, my attempt to import in the end was to import form plees-tracker.csv stored on phone storage not nextcloud, to make sure the potential permission problem is not at play.","files":null}]}